src/main/java/com/olgaboiar/mint/RoutesConfiguration.java

```diff
@@ -0,0 +1,59 @@
+package com.olgaboiar.mint;
+
+import com.olgaboiar.mint.handlers.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class RoutesConfiguration implements IRoutesConfiguration {
+    ArrayList<Route> allRoutes;
+
+    public RoutesConfiguration() {
+        createRoutes();
+    }
+
+    enum Method {
+        GET,
+        HEAD,
+        POST,
+        PUT,
+        OPTIONS
+    }
+
+    @Override
+    public void createRoutes() {
+        allRoutes = new ArrayList<> ();
+        allRoutes.add(new Route("/simple_get", new HashMap<String, IHandler>() {{
+            put(Method.GET.toString(), new RouteHandler());
+            put(Method.HEAD.toString(), new HeadHandler());
+        }}));
+        allRoutes.add(new Route("/method_options", new HashMap<String, IHandler>() {{
+            put(Method.GET.toString(), new RouteHandler());
+            put(Method.HEAD.toString(), new HeadHandler());
+            put(Method.OPTIONS.toString(), new OptionsHandler());
+        }}));
+        allRoutes.add(new Route("/method_options2", new HashMap<String, IHandler>() {{
+            put(Method.GET.toString(), new RouteHandler());
+            put(Method.HEAD.toString(), new HeadHandler());
+            put(Method.OPTIONS.toString(), new OptionsHandler());
+            put(Method.PUT.toString(), new NotAllowedHandler());
+            put(Method.POST.toString(), new NotAllowedHandler());
+        }}));
+        allRoutes.add(new Route("/get_with_body", new HashMap<String, IHandler>() {{
+            put(Method.HEAD.toString(), new HeadHandler());
+            put(Method.OPTIONS.toString(), new OptionsHandler());
+        }}));
+        allRoutes.add(new Route("/index.html", new HashMap<String, IHandler>() {{
+            put(Method.GET.toString(), new FileHandler());
+            put(Method.HEAD.toString(), new HeadHandler());
+        }}));
+        allRoutes.add(new Route("/redirect", new HashMap<String, IHandler>() {{
+            put(Method.GET.toString(), new RedirectHandler("http://0.0.0.0:5000/simple_get"));
+        }}));
+    }
```

Comment:
As discussed, `yaml` could be a good alternative for configuration data.

Reply:
used in https://github.com/olgaboiar/MintHTTPServer/pull/10/commits/bf214a872cff5e71a19f3205a6533a2a0985c765